Bottom: ce892ed9899555fb59e9db7ed56154782a9b6f6c
Top:    d61b47cd1fdfb4ea3f2f5672ed1f52dcac23b832
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-07-23 15:13:13 -0500

Rework check for first install

Create a fact for nexus_first_install rather than checking stdout
of a command in several places.



---
diff --git a/roles/nexus3/tasks/main.yml b/roles/nexus3/tasks/main.yml
index 1cfa7111..0e570bf4 100644
--- a/roles/nexus3/tasks/main.yml
+++ b/roles/nexus3/tasks/main.yml
@@ -32,7 +32,7 @@
     - nuget-group
     - nuget-hosted
     - nuget.org-proxy
-  when: nexus_data_dir_contents.stdout == "" and nexus_delete_default_repos
+  when: nexus_first_install and nexus_delete_default_repos
 
 - name: Deleting default blobstore
   include: delete_blobstore_each.yml
@@ -48,7 +48,7 @@
     - name: "{{ nexus_blob_names.mvn.blob }}"
     - name: "{{ nexus_blob_names.gitlfs.blob }}"
     - name: "{{ nexus_blob_names.yum.blob }}"
-  when: nexus_data_dir_contents.stdout == "" and nexus_delete_default_blobstore
+  when: nexus_first_install and nexus_delete_default_blobstore
 
 - include: setup_ldap_each.yml
   with_items: "{{ ldap_connections }}"
